src/main/java/htsjdk/samtools/util/IntervalListWriter.java

```diff
@@ -67,23 +67,19 @@ public IntervalListWriter(final  Path path, final SAMFileHeader header) {
      */
     public void write(final Interval interval) throws IOException {
         out.write(interval.getContig());
-        out.write('\t');
+        out.write(TAB);
         out.write(Integer.toString(interval.getStart()));
-        out.write('\t');
+        out.write(TAB);
         out.write(Integer.toString(interval.getEnd()));
-        out.write('\t');
+        out.write(TAB);
         out.write(interval.isPositiveStrand() ? '+' : '-');
```

Comment:
Perhaps use `Strand.POSITIVE` and `Strand.NEGATIVE` while you're here?

Reply:
Done

Reply:
I added a new method Interval.getStrand() that returns a Strand object that knows its string representation.